barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplemini.hpp

```diff
@@ -286,37 +285,21 @@ template <typename Curve> class ShpleminiVerifier_ {
 
         // Compute 1/(z − r), 1/(z + r), 1/(z + r²), … , 1/(z + r²⁽ⁿ⁻¹⁾)
         // These represent the denominators of the summand terms in Shplonk partially evaluated polynomial Q_z
-        const std::vector<Fr> inverse_vanishing_evals = ShplonkVerifier::compute_inverted_gemini_denominators(
+        std::vector<Fr> inverse_vanishing_evals = ShplonkVerifier::compute_inverted_gemini_denominators(
```

Comment:
this also looks like it can stay const to me

Reply:
indeed